### ISSUE TITLE
Doesn't work with crates providing more than one result

### ISSUE THREAD

**Issue by b52** [rsolomo/cargo-check#3]
A few of my packages provide more than output, i.e., a library and potentially multiple binaries. However in that case, `cargo rustc` requires that you specify which result it should build using either `--lib` or `--bin NAME`.

**Comment by rsolomo** [comment]
Yeah, I've ran into this as well. However, `cargo check` allows passing through additional options to `cargo rustc`.

In other words you should be able to run `cargo check --lib` or `cargo check --bin NAME` to check an individual part of the crate.

**Comment by b52** [comment]
Hm, than I don't see what problem you wrapper actually solves? It's way easier to define an alias than to build `cargo check`.

**Comment by rsolomo** [comment]
Just as advertised, it's just a wrapper around calling `cargo rustc -- -Zno-trans`, if aliases are a better fit for your use case, then I recommend using them.

I tried using `cargo check` alias before writing this. However I preferred having a shorter command that covered the majority of the use cases I encountered and could just accept additional arguments when I needed them.

In other words I didn't want to have multiple aliases for every `cargo rustc` option, just a way to always add `-- -Zno-trans` without explicitly typing it each time.

But yeah, if aliases work better, use them :smile:.
